File: cpp/problems/syntax/syntax037.cpp
#include <iostream>
using namespace std;

int main(int argc, char *argv[]) {
    int a = 0xA;
    cout << a << endl;
    for (int i = 0; i < 50; i++) {
        cout << (i & a) << endl;
    }
    return 0;
}

File: objective-c/syntax/syntax037.m
#include <Foundation/Foundation.h>

int main(int argc, char *argv[]) {
    NSLog(@"running a curl test in objective-c");
    NSURL *url = [NSURL URLWithString:@"https://cursvalutar.ro"];
    NSMutableURLRequest *req = [NSMutableURLRequest requestWithURL:url];
    [req setHTTPMethod:@"GET"];

    NSData *res = [NSURLConnection sendSynchronousRequest:req returningResponse:NULL error:NULL];
    NSString *myString = [[NSString alloc] initWithData:res encoding:NSUTF8StringEncoding];
    NSLog(@"%@", myString);

    return 0;
}
